utilities/transactions/optimistic_transaction_test.cc

```diff
@@ -1153,7 +1086,8 @@ TEST_P(OptimisticTransactionTest, SavepointTest) {
   s = txn->Put("D", "d");
   ASSERT_OK(s);
 
-  ASSERT_OK(txn->RollbackToSavePoint());  // Rollback to 2
+  s = txn->RollbackToSavePoint();  // Rollback to 2
+  ASSERT_OK(s);
```

Comment:
Minor nit: This is a different style to what you were doing earlier ASSERT_OK(txn...)

Not sure if there was a reason for the change?

Reply:
I have made the style more consistent.